### ISSUE TITLE
Provide total statistics metrics

### ISSUE THREAD

**Issue by jan-di** [hansmi/prometheus-paperless-exporter#39]
It would be great, if the exporter also provides metrics that represent the statistics from the UI:

<img width="383" alt="grafik" src="https://github.com/user-attachments/assets/22346348-4785-4108-995b-f306e44e04a1">

- Total count of documents/documents that are marked as new/characters
- Statistics about file extensions

**Comment by hansmi** [comment]
I had a quick look at what's necessary/possible. Statistics aren't part of the [documented REST API](https://docs.paperless-ngx.com/api/).

* "Documents in inbox" needs the "inbox" flag from tags to be exported, then it can be reproduced in PromQL (`sum(paperless_tag_document_count)` filtered by inbox flag).

* "Total documents" is possible via the API. Either by iterating (see below) or requesting `/api/documents/` which has the total count (currently not implemented by the [client code](https://pkg.go.dev/github.com/hansmi/paperhooks@v0.0.11/pkg/client).

* "Character count" would require iterating over all documents. The underlying implementation for the statistics is [a single SQL query summing the character count from all documents](https://github.com/paperless-ngx/paperless-ngx/blob/702ab1b77aa643a508702d180efe3c19aced50ce/src/documents/views.py#L1452-L1458). Depending on the scrape interval and document count the metric would be rather expensive. Would have to be behind a flag.

* The break-down by document type is also [a single SQL query grouping by MIME type](https://github.com/paperless-ngx/paperless-ngx/blob/702ab1b77aa643a508702d180efe3c19aced50ce/src/documents/views.py#L1444-L1450). Would require document iteration as well. Not sure how useful the break-down is in metrics.

Implementation:

* [Web UI](https://github.com/paperless-ngx/paperless-ngx/tree/v2.13.2/src-ui/src/app/components/dashboard/widgets/statistics-widget)
* [Internal, undocumented API](https://github.com/paperless-ngx/paperless-ngx/blob/702ab1b77aa643a508702d180efe3c19aced50ce/src/documents/views.py#L1382-L1484)

**Comment by hansmi** [comment]
The "inbox" tag flag is reported as a metric with https://github.com/hansmi/prometheus-paperless-exporter/commit/df7648d9810e3f2d4eddb59a92f5a569af570d7a.

"Total documents" will/would require changes to the underlying [Paperless-ngx client library](https://pkg.go.dev/github.com/hansmi/paperhooks/pkg/client). @jan-di, do you have a usecase other than "it would be great"?

**Comment by jan-di** [comment]
No, actually not. Its just for measuring growth as I thought its just an API that can be piped through to metrics.
If its that much work, its not worth it. 

I'm instead thinking of providing a docs MR to paperless, maybe the statistics API is already official but just not documented. If its assured that it is stable, this would make this issue much easier.

**Comment by hansmi** [comment]
@jan-di, inspired by #41 I've added a document count in https://github.com/hansmi/prometheus-paperless-exporter/commit/071024d82e53aba5466b0b64e99e11821ed87cb8. It will be part of the next release.

In summary that makes the implementation of what's done easily without additional documented APIs complete (c.f. https://github.com/hansmi/prometheus-paperless-exporter/issues/39#issuecomment-2452456176).
